README.md

```diff
@@ -224,6 +225,13 @@ testcases:
     assertions:
     - result.code ShouldEqual 0
     - result.systemout ShouldEqual "foo bar"
+
+- name: foobar
+  steps:
+  - script: echo '{{.aString}}'
+    info: value of aString is {{.aString}}
+    assertions:
+    - result.systemoutjson.foo ShouldContainSubstring bar
```

Comment:
ShouldEqual {"foo": "bar"} ?

Reply:
it's result.systemoutjson.foo and not result.systemoutjson used.
btw, it's replaced by ShouldEqual instead of  ShouldContainSubstring